GNUmakefile

```diff
@@ -244,7 +244,13 @@ generate-structs: ## Update generated code
 proto:
 	@echo "--> Generating proto bindings..."
 	@for file in $$(git ls-files "*.proto" | grep -E -v -- "vendor\/.*.proto|demo\/.*.proto"); do \
-		protoc -I . -I $(shell go env GOPATH)/src --go_out=plugins=grpc:. $$file; \
+		protoc -I . --go_out=plugins=grpc\
+	,Mplugins/shared/hclspec/hcl_spec.proto=github.com/hashicorp/nomad/plugins/shared/hclspec\
```

Comment:
This looks good - but where is `,M<path>=<package>` flag documented?  It would be nice to link to it here. If not, let's add a comment here about it, and include the need to maintain this list.